src/node_crypto.cc

```diff
@@ -1644,9 +1743,12 @@ void SSLWrap<Base>::VerifyError(const FunctionCallbackInfo<Value>& args) {
     x509_verify_error = SSL_get_verify_result(w->ssl_);
   }
 
-  if (x509_verify_error == X509_V_OK)
+  if ((x509_verify_error == X509_V_OK) && CheckWhiteList(w->ssl_))
```

Comment:
Why the parens?  Apropos the function name, I'd name it something like `IsWhitelistedServerCertificate()`, it makes it a little clearer what it does.